BREAKING.md

```diff
@@ -1,3 +1,336 @@
+
+
+#### Searchbar
+
+##### Show Cancel Button
+
+The `show-cancel-button` property of the searchbar no longer accepts boolean values. Accepted values are strings: `"focus"`, `"always"`, `"never"`.
+
+```html
+<ion-searchbar show-cancel-button>
+<ion-searchbar show-cancel-button="true">
+<ion-searchbar show-cancel-button="false">
+```
+
+becomes
+
+```html
+<ion-searchbar show-cancel-button="focus">
+<ion-searchbar show-cancel-button="focus">
+<ion-searchbar show-cancel-button="never">
+```
+
+See the [Searchbar documentation](https://ionicframework.com/docs/api/searchbar#properties) for more information.
+
+##### Inputmode
+
+The `inputmode` property for `ion-searchbar` now defaults to `undefined`. To get the old behavior, set the inputmode property to `"search"`.
+
+
+#### Segment
+
+<!-- TODO https://gist.github.com/brandyscarney/e6cfe43c359bb2c932e12f8d615e1669 -->
+
+
+#### Skeleton Text
+
+The `width` property has been removed in favor of using CSS styling.
+
+
+#### Split Pane
+- Converted to use [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM).
+- Removed the `main` attribute, use `content-id` instead.
+
+  ```html
+  <ion-split-pane>
+    ...
+    <div main>...</div>
+  </ion-split-pane>
+  ```
+
+  becomes
+
+  ```html
+  <ion-split-pane content-id="main-content">
+    ...
+    <div id="main-content">...</div>
+  </ion-split-pane>
+  ```
+
+#### Toast
+
+The close button properties (`showCloseButton` and `closeButtonText`) have been removed, use the `buttons` array instead with `role: 'cancel'`. See the [usage documentation](https://ionicframework.com/docs/api/toast#usage) for more information.
+
+```javascript
+async presentToast() {
+  const toast = await this.toastController.create({
+    message: 'Your settings have been saved.',
+    showCloseButton: true,
+    closeButtonText: 'Close'
+  });
+  toast.present();
+}
+```
+
+becomes
+
+```javascript
+async presentToast() {
+  const toast = await this.toastController.create({
+    message: 'Your settings have been saved.',
+    buttons: [
+      {
+        text: 'Close',
+        role: 'cancel',
+        handler: () => {
+          console.log('Close clicked');
+        }
+      }
+    ]
+  });
+  toast.present();
+}
+```
+
+### Colors
+
+The default Ionic colors have been updated to the following:
+
+```
+primary:         #3880ff
+secondary:       #3dc2ff
+tertiary:        #5260ff
+success:         #2dd36f
+warning:         #ffc409
+danger:          #eb445a
+light:           #f4f5f8
+medium:          #92949c
+dark:            #222428
+```
+
+`primary`, `light` and `dark` have not changed. The contrast color for `warning` has been updated to `#000`.
+
+This will only be a breaking change in your app if you are not using one of our starters & not overriding the defaults. If you are overriding the defaults already these will need to be manually updated if desired.
+
+
+### Events
```

Comment:
Should we note that this is Angular only?

Reply:
Done